pkg/scheduler/framework/plugins/noderesources/least_allocated.go

```diff
@@ -63,25 +64,46 @@ func (la *LeastAllocated) ScoreExtensions() framework.ScoreExtensions {
 }
 
 // NewLeastAllocated initializes a new plugin and returns it.
-func NewLeastAllocated(_ runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error) {
+func NewLeastAllocated(laArgs runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error) {
+	args := kubeschedulerconfig.NodeResourcesLeastAllocatedArgs{}
+
+	if err := framework.DecodeInto(laArgs, &args); err != nil {
```

Comment:
Do not use `DecodeInto` for in-tree plugins. See other plugins for reference.

Reply:
done